docs/data/catalog/operations/aws/cost-and-usage-report.md

```diff
@@ -0,0 +1,178 @@
+# AWS Cost and Usage Report
+
+Dataset describing how much was spent on AWS services by CDS.
+
+Each row describes the cost of using a particular AWS service (i.e., a line item) within a billing period.
+
+This dataset is represented in [Superset](https://superset.cdssandbox.xyz/superset/welcome/) as the Physical dataset [`cost_usage_report_by_account`](https://superset.cdssandbox.xyz/explore/?datasource_type=table&datasource_id=68). All of the Virtual datasets in the "Operations / AWS / Cost and Usage" group are derived from it.
+
+**Keywords:** AWS, Amazon, cost, usage, cost, fees
+
+## Provenance
+
+This dataset is extracted daily from the [Cost and Usage Report 2.0 (CUR 2.0)](https://docs.aws.amazon.com/cur/latest/userguide/table-dictionary-cur2.html) table in [AWS Data Exports](https://docs.aws.amazon.com/cur/latest/userguide/what-is-data-exports.html). Additional information about business units are joined in the pipeline using the `tag_env` field.
+
+More documentation on the pipeline can be found [here](../../../pipelines/operations/aws/cost-and-usage-report.md).
+
+* **Updated:** Daily
+* **Steward:** Platform Core Services
+* **Contact:** [Pat Heard](mailto:pat.heard@cds-snc.ca)
```

Comment:
Another small adjustment to point to my email (this one is just an alias since people kept using it 😅 )
```suggestion
* **Contact:** [Pat Heard](mailto:patrick.heard@cds-snc.ca)
```